docs/index.md

```diff
@@ -1,21 +1,21 @@
 The AEA framework is a Python ecosystem for autonomous economic agent (AEA) development. 
 
-It equips developers with a quick and efficient set of tools for building autonomous economic agents. 
+It equips developers with a simple to understand and efficient set of tools for building autonomous economic agents. 
 
 The framework is super modular, easily extensible, and highly composable. It is ideal for reinforcement learning scenarios. 
 
-The AEA framework is similar to many popular web frameworks.
+The AEA framework attempts to make agent development as straightforward as many popular web frameworks.
 
 
 ## Our vision
 
 The AEA framework has two focused commercial roles.
 
-### Open source company
+### Open source technology
 
-We are creating infrastructure for external to use to build their own solutions. 
+We are creating infrastructure for agent developers to use to build their own solutions. 
 
-AEA users are:
+AEA users are, among others:
 
 * Data scientists.
```

Comment:
Some suggestions for other categories:

- Students/Academics
- AI/ML Researchers 
- Crypto passionates
- Indie developers